src/componenten/fruitcounter/Fruitcounter.jsx

```diff
@@ -0,0 +1,18 @@
+import './Fruitcounter.css'
+
+// eslint-disable-next-line react/prop-types
+function Fruitcounter({ img,title, counter, increase, decrease }) {
+    return (
+        <article >
+            <span>
+            <img src={img} alt={title} />
+                </span>
+            <h2>{title}</h2>
+            <button type="button" onClick={decrease}>-</button>
```

Comment:
```suggestion
            <button type="button" onClick={decrease} disabled={counter === 0}>-</button>
```
Dan is het explicieter wat je wilt en wordt ook het disabled attribuut gebruikt waarvoor die bedoeld is 😉 
Dan zou ook je `counter > 0` checks uit die decrease functies kunnen.